plugins/AudioFileProcessor/AudioFileProcessor.h

```diff
@@ -77,6 +63,17 @@ class AudioFileProcessor : public Instrument
 
 	gui::PluginView* instantiateView( QWidget * _parent ) override;
 
+	Sample const & getSample() const { return m_sample; }
+
+	FloatModel & getAmpModel() { return m_ampModel; }
+	FloatModel & getStartPointModel() { return m_startPointModel; }
+	FloatModel & getEndPointModel() { return m_endPointModel; }
+	FloatModel & getLoopPointModel() { return m_loopPointModel; }
+	BoolModel & getReverseModel() { return m_reverseModel; }
+	IntModel & getLoopModel() { return m_loopModel; }
+	BoolModel & getStutterModel() { return m_stutterModel; }
+	ComboBoxModel & getInterpolationModel() { return m_interpolationModel; }
```

Comment:
We probably wouldn't use `get` as a prefix here.

Reply:
Adjusted with commit 1aa99e2a802.